### PR TITLE
Revises preamble

### PR DESCRIPTION
**Pull request by helrond**
Tried to bring in some of the recent changes to the preamble. Comments welcome!

### CODE REVIEW

**Review by HaSistrunk — COMMENTED** [review]
I really like the changes, but it could use a few tweaks to avoid too much repetition of "the values" and "these values."

File: values.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
...
+The values of the Rockefeller Archive Center's Digital Program inform how we work together as a team, as well as how we interact with our RAC colleagues, researchers, donors, and other external partners. These values support and inform our strategic goals and provide a standard by which we measure our daily work. We articulate these values in order to be _a part of_ a community of practice that engages technology from a basis of values and care, not to set us _apart from_ our colleagues, researchers, donors, or other external partners.
+
+We respect and contribute to a vibrant culture based on these values, and raise concerns clearly when these values are not being followed by members of the Digital Programs team. These values are reviewed on an annual basis and revised as necessary.
```
Comment (bonniegee) [inline values.md]: I'm not sure I agree with adding "by members of the Digital Programs team." I think these values influence our collaborative projects, so I'm afraid this is too limiting. But I may be overthinking it!

Comment (helrond) [reply values.md]: My thought was that it's tough for us to hold people accountable to values which they have not shaped and may not agree with. So, while I do think that these values should guide our behavior in collaborative projects, I don't know if we can demand that everyone engaged in those projects also adhere to them.

**Comment by helrond** [comment]
OK, I made some changes based on @HaSistrunk's comments.

**Comment by p-galligan** [comment]
I like the emphasis on a part of and a part from. Looks good to me.

**Review by p-galligan — APPROVED** [review]

**Pull request merged**